remaining packer content (#13285)
hashicorp/packer: website/content/docs/post-processors/shell-local.mdx

---
description: |

>   The `shell-local` post-processor starts a local shell so that you can automate post-build actions after Packer builds artifacts.

Reviewer: ```suggestion
  The `shell-local` post-processor starts a local shell, letting you automate post-build actions after Packer builds your artifacts.
```
